src/components/MarkdownRender.vue

```diff
@@ -62,11 +30,132 @@ export default {
           mode: `markdown`,
           text,
         }),
-      }).then(r => r.text());
-      o.innerHTML = res;
-    },
-    );
+      });
+      const parsedText = await res.text();
+      o.innerHTML = parsedText;
+      // localStorage.setItem(`Rendered-Markdown`, parsedText); 
+      /*  Set the Parsed text to local Storage to 
+       *    avoid making multiple API Requests    */
+      headingFilter();
+    })();
+
+    // // Retrieve localStorage parsedText - for testing
+    // const parsedText = localStorage.getItem(`Rendered-Markdown`);
+    // o.innerHTML = parsedText;
+
+
+    // Removing |== ==| 
+    const headingFilter = () => {
```

Comment:
i think its worth mentioning that this might cause unintended behavior since you dont check if its surrounded by |== ==|
```
MarkdownRender.vue?f786:54 Uncaught (in promise) TypeError: Failed to execute 'removeChild' on 'Node': parameter 1 is not of type 'Node'.
```
this is the error you get if you dont check it
instead, i think its better to change the markdown file in the git itself. ill resolve this conflict myself.

other than that, you did a fantastic job and i thank you a lot

Reply:
lol all the h2's had it as a child node at same indexes, so i didnt check it ;p

also tysm, lemme noe if i can help more :))